package-lock.json

```diff
@@ -4555,7 +4555,7 @@
                 },
                 "wrappy": {
                     "version": "1.0.2",
-                    "resolved": false,
+                    "resolved": "",
```

Comment:
looks like churn, don't actually need to commit this flip-flop

Reply:
Curating what we commit from package-lock.json is a huge pain though, it's all whatever npm decides to do

Reply:
yeah, though it's easy for cases like this, where the only meaningful change is 1 line. Up to you though